test/server.test.js

```diff
@@ -35,11 +35,17 @@ test('listen should accept stringified number port', t => {
 })
 
 test('listen should accept log text resolution function', t => {
-  t.plan(1)
+  t.plan(2)
 
   const fastify = Fastify()
   t.teardown(fastify.close.bind(fastify))
-  fastify.listen({ port: '1234', listenTextResolver: () => 'hardcoded text' }, (err) => {
+  fastify.listen({
+    port: '1234',
+    listenTextResolver: () => {
```

Comment:
I forgot the address argument, could we assert it?

Reply:
added